modules/telemetry/v1/data_block_enums.py

```diff
@@ -0,0 +1,101 @@
+from enum import IntEnum
+
+
+class DataBlockSubtype(IntEnum):
+    """Lists the subtypes of data blocks that can be sent in Version 1 of the packet encoding format."""
+
+    DEBUG_MESSAGE = 0x00
+    ALTITUDE = 0x01
+    TEMPERATURE = 0x02
+    PRESSURE = 0x03
+    ACCELERATION = 0x04
+    ANGULAR_VELOCITY = 0x05
+    GNSS_LOCATION = 0x06
+    GNSS_METADATA = 0x07
+    HUMIDITY = 0x08
+    STATUS = 0x09
+    RESERVED = 0xFF
```

Comment:
These two should be removed. There is no status data block in v1 and we also shouldn't be marking "reserved" as a subtype.